### PR TITLE
Mostly static pages

### PR DESCRIPTION
**Pull request by thucpt20**

### CODE REVIEW

**Comment by thucpt20** [comment]
Screenshot from 2019-09-26 11-12-39

**Comment by thucpt20** [comment]
ready

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,2 +1,20 @@
 class ApplicationController < ActionController::Base
+  before_action :language
+
+  private
+  def language
+    if params[:locale]
+      if I18n.available_locales.include?(params[:locale].to_sym)
+        I18n.locale = params[:locale]
+      else
+        flash[:notice] = params[:locale] << " is not support!"
```
Comment (vanvtt-0952) [inline app/controllers/application_controller.rb]: `" is not support!"` i18n chú ơi

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<ul>
+  <li><%= link_to t(".home"), root_path %></li>
+  <li><%= link_to t(".about"), about_path %></li>
+  <li><%= link_to t(".contact"), contact_path %></li>
+  <li><%= link_to t(".help"), help_path %></li>
+  <li><%= link_to "English", Settings.english.to_h %></li>
```
Comment (vanvtt-0952) [inline app/views/layouts/_header.html.erb]: I18n nè ` "English"`

**Comment by vanvtt-0952** [comment]
merged
fix comment rồi mợt đi nhé em

**Pull request merged**